src/js/edu-benefits/5495/config/form.js

```diff
@@ -0,0 +1,56 @@
+// import _ from 'lodash/fp';
+
+// import fullSchema5495 from 'vets-json-schema/dist/dependents-change-of-program-schema.json';
```

Comment:
Help. `change-of-program-schema.json`, `dependents-benefits-schema.json`, and now `dependents-change-of-program-schema.json`

@lihanli is it unthinkable to change the names to `form-22-5490-schema.json` before we launch the 1990e and 5490? It's *so confusing* (or maybe my brain is just too small to hold all of these variations)